lib/Charts/Tooltip.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-import d3 from 'd3';
+import * as d3Collection from 'd3-collection';
+import * as d3Selection from 'd3-selection';
+import * as d3Transition from 'd3-transition'; // eslint-disable-line no-unused-vars
```

Comment:
I think it's better to be explicit about what we're importing - instead of `import *`, can you use change this to eg.
```import {selection as d3Selection} from `d3-selection```?